tests/baselines/reference/privateNameInObjectLiteral-1.errors.txt

```diff
@@ -0,0 +1,10 @@
+tests/cases/conformance/classes/members/privateNames/privateNameInObjectLiteral-1.ts(2,5): error TS18016: Private names are not allowed outside class bodies.

```

Comment:
how is this test different from `privateNameInObjectLiteral` ?

Reply:
stray file